tests/dummy/app/templates/components/main-navigation.hbs

```diff
@@ -1,26 +1,23 @@
-<nav class="navbar navbar-default navbar-fixed-top">
-  <div class="container-fluid">
-    <div class="navbar-header">
-      {{#link-to 'index' classNames='navbar-brand'}}
-        Home
-      {{/link-to}}
-    </div>
-    <ul class="nav navbar-nav navbar-left">
-      {{#link-to 'protected' tagName='li'}}
-        <a>Protected Page</a>
-      {{/link-to}}
-      {{#link-to 'auth-error' tagName='li'}}
-        <a>Auth Error Page</a>
-      {{/link-to}}
-    </ul>
-    {{! display logout button when the session is authenticated, login button otherwise }}
-    {{#if session.isAuthenticated}}
-      <a {{action 'logout'}} class="btn btn-danger navbar-btn navbar-right">Logout</a>
-      {{#if sessionAccount.account}}
-        <p class="navbar-text pull-right">Signed in as {{sessionAccount.account.name}}</p>
-      {{/if}}
-    {{else}}
-      <a {{action 'login'}} class="btn btn-success navbar-btn navbar-right">Login</a>
+<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  {{#link-to "index" classNames="navbar-brand"}}
+    Home
+  {{/link-to}}
+
+  <ul class="navbar-nav mr-auto">
+    {{#link-to "protected" tagName="li" classNames="nav-item"}}
+      <a class="nav-link">Protected Page</a>
+    {{/link-to}}
+    {{#link-to "auth-error" tagName="li" classNames="nav-item"}}
+      <a class="nav-link">Auth Error Page</a>
+    {{/link-to}}
+  </ul>
+  {{! display logout button when the session is authenticated, login button otherwise }}
+  {{#if session.isAuthenticated}}
+    <a {{action "logout"}} class="btn btn-danger navbar-btn navbar-right">Logout</a>
```

Comment:
I think this one could use the `.text-white` color utility class to make the button more readable and to not regress visually from the current layout:

|this branch|master|
|-|-|
|<img width="111" alt="screen shot 2018-04-18 at 14 49 39" src="https://user-images.githubusercontent.com/8811742/38932883-c0283b74-4317-11e8-8173-dec8bd89dfc3.png">|<img width="90" alt="screen shot 2018-04-18 at 14 49 46" src="https://user-images.githubusercontent.com/8811742/38932902-c8183c8a-4317-11e8-8188-b6e1f6671dc3.png">|

See also the color utility class doc here: https://getbootstrap.com/docs/4.1/utilities/colors/#color